src/constants.js

```diff
@@ -174,6 +174,11 @@ export const emotesCategoryIds = {
   FRECENTS: 'frecents',
 };
 
+export const tipIds = {
+  EMOTE_MENU_ALT_FAVORITE_EMOTE: 'clickAltToFavoriteEmote',
```

Comment:
why do we need key codes in these identifiers? just name them what they are for?

```
emoteMenuFavorite
emoteMenuPreventClose
```